tests/functional/api/test_query.py

```diff
@@ -0,0 +1,12 @@
+from ape import accounts, chain
+
+
+def test_basic_query(test_provider):
+    a = accounts.test_accounts[0]
+    a.transfer(a, 100)
```

Comment:
Calling `test_provider.mine()` is probably a better way to advance the blockchain

Reply:
I think you can ever just do `test_provider.mine(3)` to replace all 3 lines